### PR TITLE
az-config: add German translation

### PR DESCRIPTION
**Pull request by sidyes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.de/common/az-config.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Verwalten der der Azure CLI-Konfiguration.
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
+> Mehr Informationen: <https://learn.microsoft.com/cli/azure/config>.
```
Comment (kbdharun) [inline pages.de/common/az-config.md:5]:
```suggestion
> Weitere Informationen: <https://learn.microsoft.com/cli/azure/config>.
```

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
@marchersimon @pixelcmtd I have a doubt is snake case necessary in these command arguments?

**Comment by pixelcmtd** [comment]
@kbdharun I'm not sure if I understand your question correctly, do you mean underscores? Either way works.

File: pages.de/common/az-config.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# az config
+
+> Verwalten der der Azure CLI-Konfiguration.
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:3]:
```suggestion
> Verwalten der Azure CLI-Konfiguration.
```

File: pages.de/common/az-config.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+> Teil der `Azure CLI`, der Befehlszeilenschnittstelle von Azure.
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:4]:
```suggestion
> Teil von `azure-cli`.
```

File: pages.de/common/az-config.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+> Weitere Informationen: <https://learn.microsoft.com/cli/azure/config>.
+
+- Rufen Sie alle Konfigurationen ab:
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:7]:
```suggestion
- Rufe alle Konfigurationen ab:
```

File: pages.de/common/az-config.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+`az config get`
+
+- Rufen Sie alle Konfigurationen in einem Abschnitt ab:
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:11]:
```suggestion
- Rufe alle Konfigurationen in einer Sektion ab:
```

File: pages.de/common/az-config.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+`az config get {{sektionsname}}`
+
+- Legen Sie eine Konfiguration fest:
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:15]:
```suggestion
- Setze eine Konfiguration:
```

File: pages.de/common/az-config.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+`az config set {{konfigurationsname}}={{wert}}`
+
+- Heben Sie eine Konfiguration auf:
```
Comment (pixelcmtd) [inline pages.de/common/az-config.md:19]:
```suggestion
- Hebe eine Konfiguration auf:
```

**Comment by sidyes** [comment]
Can you approve and mehre @pixelcmtd ?

**Review by pixelcmtd — APPROVED** [review]
I'm sorry I might not be able to mehre ^^

**Comment by kbdharun** [comment]
> @kbdharun I'm not sure whether I understand your question correctly, do you mean underscores? Either way works.

Yes, I meant underscores. Thanks for the clarification.

**Pull request merged**